src/components/molecules/Settings/Section/index.tsx

```diff
@@ -15,47 +14,35 @@ const Section: React.FC<Props> = ({ title, actions, children }) => {
       {title && (
         <>
           <SectionHeader>
-            <Text
-              size="l"
-              weight="normal"
-              color={theme.main.strongText}
-              otherProperties={{ flex: 1 }}>
+            <Text size="l" weight="normal" color={theme.main.strongText}>
               {title}
             </Text>
             {actions}
           </SectionHeader>
           <Divider />
         </>
       )}
-      <SectionItem>
-        <Field>{children}</Field>
-      </SectionItem>
+      <SectionItem>{children}</SectionItem>
     </Wrapper>
   );
 };
 
-const Wrapper = styled.div`
-  max-width: 100%;
-`;
+const Wrapper = styled.div``;
```

Comment:
This doesn't need to be a styled component

Reply:
ok, thanks